sources/engine/Stride.Engine/Engine/TransformComponent.cs

```diff
@@ -145,16 +145,16 @@ public Vector3 RotationEulerXYZ
                 float yz = rotation.Y * rotation.Z;
                 float xw = rotation.X * rotation.W;
 
-                rotationEuler.Y = (float)Math.Asin(2.0f * (yw - zx));
+                rotationEuler.Y = MathF.Asin(2.0f * (yw - zx));
                 double test = Math.Cos(rotationEuler.Y);
```

Comment:
Found this one leftover

Reply:
If you're referring to the ``double test = ...`` yes, I didn't end up changing stuff that used the double precision just to stay safe but this one is fine.